gcloud/pubsub/api.py

```diff
@@ -52,3 +52,49 @@ def list_topics(page_size=None, page_token=None,
 
     path = '/projects/%s/topics' % project
     return connection.api_request(method='GET', path=path, query_params=params)
+
+
+def list_subscriptions(page_size=None, page_token=None, topic_name=None,
+                       project=None, connection=None):
+    """List subscriptions for a given project.
+
```

Comment:
Can you include the resource->method it maps to in
https://www.googleapis.com/discovery/v1/apis/pubsub/v1beta2/rest

It seems they are

```
pubsub.projects.subscriptions.list
pubsub.projects.topics.subscriptions.list
```